README.md

```diff
@@ -1,16 +1,15 @@
-# charmed-kafka-snap
+# Charmed Kafka snap repository.
 
-Charmed Kafka snap repository.
-
-This snap is based on the kafka snap: https://git.launchpad.net/kafka-snap/
+[![CharmHub Badge](https://snapcraft.io/charmed-kafka/badge.svg)](https://snapcraft.io/charmed-kafka)
+[![Release](https://github.com/canonical/charmed-kafka-snap/actions/workflows/publish.yaml/badge.svg)](https://github.com/canonical/charmed-kafka-snap/actions/workflows/publish.yaml)
 
```

Comment:
[![Update Available](https://github.com/canonical/charmed-kafka-snap/actions/workflows/on_kafka_update_available.yaml/badge.svg)](https://github.com/canonical/charmed-kafka-snap/actions/workflows/on_kafka_update_available.yaml)
```suggestion
[![Update Available](https://github.com/canonical/charmed-kafka-snap/actions/workflows/on_kafka_update_available.yaml/badge.svg)](https://github.com/canonical/charmed-kafka-snap/actions/workflows/on_kafka_update_available.yaml)
```

Reply:
Uhm, I would skip this one if you don't mind, as we should actually remove that workflow. That workflow was actually looking upstream, but we now build from source.